include/triton/core/tritonbackend.h

```diff
@@ -34,19 +34,19 @@ extern "C" {
 #endif
 
 #ifdef _COMPILING_TRITONBACKEND
```

Comment:
Are all the changes in this file just from the formatter?

Reply:
All the changes from this commit 292cfe2649d5ebb3b are from the formatter. I can quickly share a PR for (DLIS-1738)[https://jirasw.nvidia.com/browse/DLIS-1738] to simplify this change. I think I have all the code ready, just need to make sure all the repos are formatted.